_install-and-configure/plugins.md

```diff
@@ -303,6 +303,22 @@ _<sup>1</sup>Dashboard Notebooks was merged in to the Observability plugin with
 _<sup>2</sup>Performance Analyzer is not available on Windows._
 
 
+### Downloading plugin distributions for offline installation
+
+Each bundled plugin distribution can be downloaded and installed from a [zip file](#install-a-plugin-from-a-zip-file) offline using the official `plugins` repository URL template:
```

Comment:
"from an offline zip file"?

Reply:
The file itself is hosted online. The installation happens offline.

Reply:
I'll change the word order so it's clear.